process-automation/latest/model/projects.md

```diff
@@ -65,6 +65,8 @@ To release a project:
 
 Selecting the **View Releases** option for a project displays the version history for the project.
 
+You can restore your project to a previous release by selecting the three dots to the right of the release you want to restore, and then selecting **Restore this Release**.
```

Comment:
```suggestion
We can restore your project to a previous release by selecting the three dots to the right of the release you want to restore, and then selecting **Restore this Release**.
```

Reply:
I wont make those changes cheers. We use the You  instead here.